### PR TITLE
gitcrypt instructions

### PR DESCRIPTION
**Pull request by davidread**
I wrote these for another repo. They are a bit verbose, but better than nothing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+2. Once you receive the file, save it on your disk e.g. /tmp/alice.asc
+
+3. Import it into your GPG keyring:
+
+       gpg --import /tmp/alice.asc
+
+4. Tell GPG that you trust the key and sign it:
+
+       gpg --edit-key "alice@cyb.org" trust
+         # 4
```
Comment (xoen) [inline README.md]: Is `4` enough? I remember having troubles months ago so I resorted to use `5` (absolute trust, I'd trust that person is who they say they are and I'd trust that person with my life and my dog).

Maybe I didn't try `4` 🤷‍♂️

File: README.md [hunk 2]
```diff
@@ -1,2 +1,65 @@
 # analytics-platform-config
 Encrypted configuration
+
+## Gitcrypt
+
+This repo contains secrets stored with gitcrypt. These need decrypting before use. You need to have your GPG key added to this repo before you are able to do this.
```
Comment (xoen) [inline README.md]: At the risk of being punched from the right side, I'd probably change `gitcrypt` to `git-crypt`, sorry a bit pedantic, don't hate me, feel free to ignore it :)

File: README.md (hunk 1 above)
Comment (davidread) [reply README.md]: You have me wondering now... That's what I recorded at the time but we can change it if we find it's wrong.

File: README.md (hunk 2 above)
Comment (davidread) [reply README.md]: Cool, I'll fix that

**Pull request merged**